rgw/v2/tests/s3_swift/reusable.py

```diff
@@ -810,12 +810,13 @@ def delete_version_object(
             if response.status_code == 204:
                 log.info("version deleted ")
                 write_key_io_info = KeyIoInfo()
-                write_key_io_info.delete_version_info(
+                # commentig the code, since it is irrelevant.
```

Comment:
Is this code not being used in any other scripts?

Reply:
@TejasC88 , one script does, bt still the code is not relevant.